src/ShallowWrapper.js

```
@@ -98,15 +91,20 @@ function validateOptions(options) {
   }
 }
 
+function getRootNode(node) {
+  if (node.nodeType === 'host') {
+    return node;
+  }
+  return node.rendered;
+}
 
-function performBatchedUpdates(wrapper, fn) {
-  const renderer = wrapper.root.renderer;
-  if (REACT155 && renderer.unstable_batchedUpdates) {
-    // React 15.5+ exposes batching on shallow renderer itself
-    return renderer.unstable_batchedUpdates(fn);
+function getAdapter(options) {
+  if (options.adapter) {
+    return options.adapter;
   }
-  // React <15.5: Fallback to ReactDOM
-  return batchedUpdates(fn);
+  const adapter = configuration.get().adapter;
+  // TODO(lmr): warn about no adapter being configured
```

Should this throw instead of warn?